x-pack/test/search_sessions_integration/tests/apps/discover/async_search.ts

```diff
@@ -131,19 +131,6 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
       expect(searchesCountBeforeRestore).to.be(searchesCountAfterRestore); // no new searches started during restore
     });
-
-    it('should should clean the search session when navigating to SQL mode, and reinitialize when navigating back', async () => {
```

Comment:
Just curious why this test was removed?

Reply:
Because the search sessions is not cleaned up for this particular test as the histogram is depicted.